osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -127,5 +124,17 @@ protected override void UpdateAfterChildren()
 
             comboDisplay.X = MovableCatcher.X;
         }
+
+        private DrawableCatchHitObject createCaughtFruit(DrawablePalpableCatchHitObject hitObject)
+        {
+            return hitObject.HitObject switch
+            {
+                Banana banana => new DrawableBanana(banana),
+                Fruit fruit => new DrawableFruit(fruit),
+                TinyDroplet tiny => new DrawableTinyDroplet(tiny),
+                Droplet droplet => new DrawableDroplet(droplet),
+                _ => null
+            };
```

Comment:
use a standard `switch` here

Reply:
What is the reason the switch statement is preferred than the switch expression?

Reply:
Some are opposed to the `x switch` syntax.